### PR TITLE
add .gitpod.yml

### PR DESCRIPTION
**Pull request by jcommelin**
Please make sure you modified the files in `src/solutions` and used the `mk_exercises.py` script to recreate the files in `src/exercises`. Otherwise your modifications will soon be automatically overwritten.

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+image: leanprovercommunity/mathlib:gitpod
+
+vscode:
+  extensions:
+    - jroesch.lean
+
+tasks:
+  - command: . /home/gitpod/.profile && scripts/get-cache.sh
```
Comment (robertylewis) [inline .gitpod.yml]: This is a `lean-liquid` script, isn't it?

Comment (jcommelin) [reply .gitpod.yml]: aah, that's a good point, I guess. I'll copy it over for now.

Comment (jcommelin) [reply .gitpod.yml]: Actually, I removed the call to that script. I don't think we have olean caches for the tutorials project, and maybe we don't need them either.

**Comment by robertylewis** [comment]
Do you want to add a link/banner/whatever to the readme so it's easier to find? Otherwise looks good, we can test it live.

**Comment by jcommelin** [comment]
Good idea. Done.

File: .gitpod.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+image: leanprovercommunity/mathlib:gitpod
+
+vscode:
+  extensions:
+    - jroesch.lean
+
+tasks:
+  - command: . /home/gitpod/.profile
```
Comment (eric-wieser) [inline .gitpod.yml]: Shouldn't this have a `leanpkg configure` and `leanpkg get-mathlib-cache`? (Or maybe just one of them)

Comment (jcommelin) [reply .gitpod.yml]: I don't know! I have no idea how gitpod works.

Comment (robertylewis) [reply .gitpod.yml]: Ah, yeah. The call to the script you deleted would have done this in `lean-liquid`. https://github.com/leanprover-community/lean-liquid/blob/master/scripts/get-cache.sh

You need at least `leanproject get-mathlib-cache`. I don't know what directory this runs in, you might need the `cd` too.

**Comment by jcommelin** [comment]
As I clearly have no idea what I'm doing, I would love to hand this over to someone who has experience with setting up repos for gitpod.

**Pull request merged**